lib/paymentrails/gateways/BatchGateway.rb

```diff
@@ -30,7 +30,18 @@ def update(batch_id, body)
     end
 
     def delete(batch_id)
```

Comment:
Please add comment on this method specifying the delete command now takes an array AND a string both

Reply:
added in https://github.com/PaymentRails/ruby-sdk/pull/52/commits/7e87f13388e14242686a9b87d65c5995d35a034f